src/main/java/com/rcv/Tabulator.java

```diff
@@ -688,6 +688,22 @@ private OvervoteDecision getOvervoteDecision(Set<String> candidateIDSet) {
 
     // CVR indexes over the cast vote records to count votes for continuing candidateIDs
     for (CastVoteRecord cvr : castVoteRecords) {
+      // see if this cvr recipient is a continuing candidate
```

Comment:
You can make this comment a little clearer. Suggestion: "If this CVR was assigned to a candidate last round and that candidate is still continuing, we can safely assume that the CVR should still be assigned to that candidate in this round."